src/commands/math/basicSymbols.js

```diff
@@ -418,9 +418,10 @@ var PlusMinus = P(BinaryOperator, function(_) {
   _.contactWeld = _.siblingCreated = _.siblingDeleted = function(opts, dir) {
     if (dir === R) return; // ignore if sibling only changed on the right
     // If the left sibling is a binary operator or a separator (comma, full stop, semicolon, colon)
+    // or an open bracket (open parenthesis, open square bracket)
     // consider the operator to be unary, otherwise binary
     this.jQ[0].className =
-      (!this[L] || this[L] instanceof BinaryOperator || ',.;:'.indexOf(this[L].ctrlSeq) >= 0 ? '' : 'mq-binary-operator');
+      (!this[L] || this[L] instanceof BinaryOperator || ',.;:(['.indexOf(this[L].ctrlSeq) >= 0 ? '' : 'mq-binary-operator');
```

Comment:
interesting. Normally this isn't possible to type, typing `(` or `[` creates a paren group not a single symbol, but they can still be rendered from LaTeX via the API or being pasted in, I guess this is the right thing to do in that situation